src/models/PrimerClientSession.ts

```diff
@@ -0,0 +1,53 @@
+export type PrimerClientSession = IPrimerClientSession;
```

Comment:
Can we change these exports to only export the interface and then export all interfaces as types in `index.tsx`?